src/scripts/main.js

```diff
@@ -1 +1,15 @@
 'use strict';
+
+const thumbs = document.getElementById('thumbs');
+const largeImg = document.getElementById('largeImg');
+
+thumbs.addEventListener('click', (ev) => {
+  const imgLink = ev.target.closest('.list-item__link');
```

Comment:
Ensure that the HTML structure consistently provides a parent with the class `.list-item__link` for the clicked element. If the structure changes or if clicks are made on elements without such a parent, `imgLink` will be `null`, and the subsequent code will not execute as expected. Consider adding a check to handle cases where `imgLink` is `null` to avoid potential errors.